doc/wakaama_features.rst

```diff
@@ -0,0 +1,213 @@
+Wakaama Features
+----------------
+
+This table shows the LwM2M features that are supported by Wakaama.
+
++----------------------------+----------------------------------------+------------------+-------------------------------+
+| Feature Type               | Feature                                | Wakaama supports | Note                          |
++============================+========================================+==================+===============================+
+| Transport                  | DTLS                                   | v_1.0            | by Tinydtls                   |
```

Comment:
Not sure what `v_1.0`/`v_1.1` means. Is this a tag? Preprocessor macro?

Reply:
Agree. That's not clear.

Reply:
Not sure how to document this.
Maybe change all `v_1.0`/`v_1.1` to `yes` as we implicitly target LwM2M version 1.1 (or 1.1.1) in Wakaama.
There is some basic (not maintained) support for implementing a LwM2M version 1.0 client.